### PR TITLE
Add `Build` tab element

### PR DESCRIPTION
**Pull request by dem4ron**
Screenshot 2022-11-08 at 11 12 05

### TODO:
- Make the header visible when visiting `/build` page

### CODE REVIEW

**Comment by iHiD** [comment]
Let's not merge this for now. That will enable us to dark-deploy without this bit.

(this might be why you've done it as a seperate PR 🙂)

File: app/views/tracks/build/show.html.haml [hunk 1]
```diff
@@ -1,6 +1,9 @@
+- track_meta_tags(@user_track)
```
Comment (ErikSchierboom) [inline app/views/tracks/build/show.html.haml]: I've snuck this in as well

**Review by ErikSchierboom — COMMENTED** [review]
@iHiD This PR is now ready for review.

One thing we need to remember is to clear the Rails cache for the header, as otherwise the build tab won't show up. What would be the most efficient way of doing that?

File: app/views/tracks/build/show.html.haml (hunk 1 above)
Comment (iHiD) [reply app/views/tracks/build/show.html.haml]: Nice

Comment (iHiD) [reply app/views/tracks/build/show.html.haml]: You'll actually need to change this, else we'll have duplicate meta-titles which hurt us from an SEO perspective.

So you'll need to manually do:

```
content_for :meta_title, "#{user_track.track_title} build page on Exercism"
```

etc (same for description, image, etc)

Comment (ErikSchierboom) [reply app/views/tracks/build/show.html.haml]: I'll then remove it and fix it in another PR (other track pages have this too)

**Review by iHiD — APPROVED** [review]
LGTM

**Pull request merged**